model/project.go

```diff
@@ -862,7 +862,9 @@ func (p *Project) BuildProjectTVPairsWithAlias(alias string) ([]TVPair, error) {
 		for _, variant := range p.BuildVariants {
 			if variantRegex.MatchString(variant.Name) {
 				for _, task := range p.Tasks {
-					if taskRegex.MatchString(task.Name) && (p.FindTaskForVariant(task.Name, variant.Name) != nil) {
+					if taskRegex.MatchString(task.Name) &&
```

Comment:
should this be an or rather than an and?

Reply:
changed to or